### PR TITLE
correct the DNS entries to have a valid root domain

### PR DESCRIPTION
**Pull request by theofpa**

### CODE REVIEW

File: content/docs/aws/aws-e2e.md [hunk 1]
```diff
@@ -208,12 +208,13 @@ kfctl apply -f kfctl_aws_cognito.v1.0.0.yaml -V
...
 That shouldn't take a long time. There shouldn't by any errors, and when ready you can validate that you can see the kubeflow namespace.
 
-At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 3 CNAME entries to it in Route53:
+At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 2 CNAME entries to it in Route53:
 
-* `platform.domain.com`
 * `*.platform.domain.com`
 * `*.default.platform.domain.com`
 
+And one A record for the root domain `platform.domain.com` to make it valid, which can be a Route53 Alias to the ALB as well. If you're not using Route53, you can point that A record anywhere.
```
Comment (Jeffwan) [inline content/docs/aws/aws-e2e.md]: Can you move this note to https://www.kubeflow.org/docs/aws/aws-e2e/#certificate-manager? 
I think most users experience the error there when they setup ACM

Comment (Jeffwan) [reply content/docs/aws/aws-e2e.md]: BTW, is `*.default.platform.domain.com` needed?

File: content/docs/aws/aws-e2e.md [hunk 2]
```diff
@@ -208,12 +208,13 @@ kfctl apply -f kfctl_aws_cognito.v1.0.0.yaml -V
 
 That shouldn't take a long time. There shouldn't by any errors, and when ready you can validate that you can see the kubeflow namespace.
 
-At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 3 CNAME entries to it in Route53:
+At this point you will also have an ALB, it takes around 3 minutes to be ready. When ready, copy the DNS name of that load balancer and create 2 CNAME entries to it in Route53:
 
-* `platform.domain.com`
 * `*.platform.domain.com`
 * `*.default.platform.domain.com`
```
Comment (Jeffwan) [inline content/docs/aws/aws-e2e.md]: Is `*.default.platform.domain.com` needed?

Comment (theofpa) [reply content/docs/aws/aws-e2e.md]: Yes, it's required for the next step, which is the inference service.

Comment (Jeffwan) [reply content/docs/aws/aws-e2e.md]: Make sense.

**Pull request merged**